Loader - Bukkit/src/me/devtec/theapi/bukkit/packetlistener/PacketHandlerModern.java

```diff
@@ -73,6 +73,9 @@ public PacketHandlerModern(boolean lateBind) {
 				hasTicked = "ac";
 				break;
 			}
+			case 20:
+				hasTicked = "ad";
+				break;
 			while (!(boolean) Ref.get(BukkitLoader.getNmsProvider().getMinecraftServer(), hasTicked))
```

Comment:
Without the "ad" field this loop is running infinitely resulting in not registering the packet listening feature

Reply:
You are right, thank you very much, sir.